### DISCUSSION TITLE
LDAP User to use

### DISCUSSION THREAD

**Discussion by agorosti** [HCL-TECH-SOFTWARE/connections-automation#207] [Q&A]
Hi,
can someone please clarify what LDAP User admin is used on the OpenLdap ?

I do have on my vars.file

ldap_nr_of_users: 10
ldap_userid: "fakeuser"
ldap_user_password: "mypassword"
ldap_user_admin_password: "mypassword"
ldap_user_mail_domain:  "connections.mydomain.com"

Is ldap admin created with the username  "fakeuser" or is created with the username "Admin" ?

I am using a ldap browser cleint to authenticate on the ldap server and check users,  but when I try to authenticate using cn=Admin, or cn=fakeuser on the ldap client to check the ldap server contents, I am unable to do it, it says that wrong credentials are used.

I am using dc=cnx,dc=pnp-hcl,dc=com  as BaseDN

**Comment by sabrina-yee** [comment]
Hi @agorosti, the default is `cn=Admin,dc=cnx,dc=pnp-hcl,dc=com` (per https://github.com/HCL-TECH-SOFTWARE/connections-automation/blob/main/roles/third_party/ldap-install/vars/main.yml#L6-L7).

**Reply by agorosti** [reply]
So, I can not understand what is happening...

I used ldapsearch -x -b "dc=cnx,dc=pnp-hcl,dc=com" -H ldap://127.0.0.1 -D "cn=Admin,dc=cnx,dc=pnp-hcl,dc=com" -W to check the ldap data, and I always had "invalid credentials" when trying to bind.     I am pretty sure I modified the vars.file before I ran the ansible scripts, with a the password I was trying to use to validate, but It did not work.

So,  I tryed to use the standard password that come as default  (password),  and it worked !!     Seems that the ansible script did not collect the changes I did at the all.yml to use a different password..... not sure if that is a bug, or I did something wrong...

**Reply by sabrina-yee** [reply]
I see your original post has `ldap_user_admin_password` set, do you have `ldap_bind_pass` set too?  The former is the password for the 1st test user (typically used as the Connections admin user), while the latter is the LDAP bind user password which I believe is what you're looking for.

**Comment by stoeps13** [comment]
Have you set ldap_bind_pass in your variables file?
